### ISSUE TITLE
Bug: "alias" option is deprecated. Use "resolve.alias" instead.

### ISSUE THREAD

**Issue by ParamagicDev** [ElMassimo/vite_ruby#15]
### Description 📖

On a new project, after going through setup steps on a new rails project, attempt to run `bundle exec vite build` and receive the following error:

```bash
bundle exec vite build
I, [2021-02-16T20:35:08.134871 #67630]  INFO -- : Building with Vite ⚡️
E, [2021-02-16T20:35:08.846910 #67630] ERROR -- : Build with Vite failed:
building for production...
✓ 0 modules transformed.

(!) "alias" option is deprecated. Use "resolve.alias" instead.
You must supply options.input to rollup
error during build:
Error: You must supply options.input to rollup
    at Graph.generateModuleGraph (/Users/konnorrogers/projects/personal/carebnb/node_modules/vite/node_modules/rollup/dist/shared/rollup.js:19184:19)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async Graph.build (/Users/konnorrogers/projects/personal/carebnb/node_modules/vite/node_modules/rollup/dist/shared/rollup.js:19148:9)
    at async rollupInternal (/Users/konnorrogers/projects/personal/carebnb/node_modules/vite/node_modules/rollup/dist/shared/rollup.js:20053:9)
    at async doBuild (/Users/konnorrogers/projects/personal/carebnb/node_modules/vite/dist/node/chunks/dep-9bcbff8e.js:46359:24)
    at async build (/Users/konnorrogers/projects/personal/carebnb/node_modules/vite/dist/node/chunks/dep-9bcbff8e.js:46305:16)
    at async CAC.<anonymous> (/Users/konnorrogers/projects/personal/carebnb/node_modules/vite/dist/node/cli.js:13985:9)
```

EDIT: After some digging im assuming this is most likely the culprit:

https://github.com/ElMassimo/vite_ruby/blob/1d859e367cc956dbb9ac1d0454dc311d8708e6b3/vite-plugin-ruby/src/index.ts#L58

**Comment by ElMassimo** [comment]
Hi Konnor, as it mentions in the comment, we are passing it in the old format as well, in case anyone had pinned an older version of Vite.

Now that Vite is officially out of beta, I'll remove this line.

Thanks for the reminder!

**Comment by ParamagicDev** [comment]
@ElMassimo No problem! I figured as much looking at the code, figured I'd file a formal issue :)

**Comment by ElMassimo** [comment]
Closed in https://github.com/ElMassimo/vite_ruby/commit/64c58656a667823b36d076055593fa954c1463f7.

Released `"vite-plugin-ruby": "^1.0.13"`.
